### PR TITLE
Remove ant build system

### PR DESCRIPTION
**Pull request by jburel**
We have not switched to gradle and we use travis to create the various executables
Those files are no longer necessary

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -9,6 +9,7 @@ out
 
 # Gradle cache
 .gradle
+gradle
```
Comment (joshmoore) [inline .gitignore]: What's creating this file?

Comment (jburel) [reply .gitignore]: If you want to test with another version of gradle than the one you have installed e.g. ``gradle wrapper --gradle-version=5.2.1``
I now personally use different conda env but this is not always the case

**Comment by dominikl** [comment]
Works fine 👍

**Comment by jburel** [comment]
nothing depends on ivy/ant anymore.

**Pull request merged**